moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -485,6 +490,8 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
     }
   }, [data, advancedSearchAnchor]);
 
+    console.log(query.query)
```

Comment:
Maybe this is leftover from testing or it is important feedback - not a big deal at all

Reply:
cleaned it up! thank you for pointing it out.